ciao-controller/instance.go

```diff
@@ -46,7 +46,7 @@ type instance struct {
 	startTime time.Time
 }
 
-func isCNCIWorkload(workload *types.Workload) bool {
+func isCNCIWorkload(workload types.Workload) bool {
```

Comment:
@markdryan another thing that would be a great clarification for me - in the go class we took, they told us to avoid passing by pointer if possible due to it complicating garbage collection and go's escape analysis. At what point do we need to worry about that?